packages/react/src/components/utils/index.tsx

```diff
@@ -27,6 +27,21 @@ export const createForwardRef = <PropType, ElementType>(
   return React.forwardRef(forwardRef);
 };
 
+export const mergeRefs = (
+  ...refs: (React.ForwardedRef<any> | React.Ref<any> | undefined)[]
+) => {
+  return (value: any) => {
+    refs.forEach(ref => {
+      if (typeof ref === 'function') {
+        ref(value)
+      } else if (ref != null) {
+        // Cast as a MutableRef so we can assign current
+        (ref as React.MutableRefObject<any>).current = value
```

Comment:
We can probably use the helper function here as well (we may need to cast as MutableRef in the helper function)